File: 02500/BOJ_2563.py
import sys
input=sys.stdin.readline

arr = [[0 for _ in range(100)] for _ in range(100)]
n = int(input())
ans = 0

for _ in range(n):
    r, c = map(int, input().split())
    for i in range(r, r+10):
        for j in range(c, c+10):
            arr[i][j] += 1
            if arr[i][j] == 1:
                ans += 1

print(ans)

File: 02500/BOJ_2563.js
const readline = require("readline");
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

let input = [];

rl.on("line", line => {
  input.push(line);
}).on("close", () => {
  const n = parseInt(input[0]);
  let ans = 0;
  let arr = Array.from(Array(100), () => Array(100).fill(0))
  for (t = 1; t <= n; t++) {
  	const tmp = input[t].split(" ").map(i => parseInt(i));
	let r = tmp[0];
	let c = tmp[1];
	for (i = r; i < r+10; i++) {
		for (j = c; j < c+10; j++) {
			if (++arr[i][j] == 1)
				ans++;
		}
	}
  }
  console.log(ans);

  process.exit();
});
